ethereum/contracts/ShutdownSwitch.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "./interfaces/IWormhole.sol";
+
+contract ShutdownStructs {
+    struct VoterEntry {
+        address key;
+        bool enabled;
+    }
+}
+
+abstract contract ShutdownSwitch {
```

Comment:
It would be nice to have an overview comment of what this class does. Something like https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/ReentrancyGuard.sol#L6-L21